tide_webform.module

```diff
@@ -165,13 +170,37 @@ function tide_webform_webform_submission_create_access(AccountInterface $account
 }
 
 /**
- * Implements hook_ENTITY_TYPE_presave().
+ * Implements hook_form_FORM_ID_alter().
  */
-function tide_webform_webform_presave(EntityInterface $entity) {
-  if ($entity->isNew()) {
-    $access_rules = $entity->getAccessRules();
```

Comment:
Setting the access rules in `hook_entity_presave` will make the form always exposed to API config even when it is created programmatically elsewhere.